views/static/press-kit.jade

```diff
@@ -30,12 +31,12 @@ block content
           //.carousel-caption=img
     a.left.carousel-control(href='#carousel', role='button', data-slide='prev')
       span.glyphicon.glyphicon-chevron-left(aria-hidden='true')
-      span.sr-only Previous
+        span.sr-only= env.t('previous')
```

Comment:
The extra tab on line 34 and 37 aren't a huge deal right now, since the carousel is commented out, but if it ever gets included back into the page, it will result in the span being nested in the span above it, so if you could remove the extra spaces, that'd be great.